### ISSUE TITLE
Feature: detect patter vs singing vs vocals by label/label#

### ISSUE THREAD

**Issue by mpogue2** [mpogue2/SquareDesk#59]
It should be possible -- all records are known as to whether they are initially patter or singing.  We could look that up in a DB somewhere.

**Comment by danlyke** [comment]
Usually the singing vs vocals have a "a", "b" (as in sides of the 45) or "v" somewhere around the label number, if they're not listed as "(vocal)" in the extended stuff. Naming is so inconsistent, and it wasn't always consistent about what went on the "a" or "b" side of the 45 (or how it was called when it was imported by whoever imported it).

This seems like something that may fall out of better filename matching.

**Comment by mpogue2** [comment]
Agreed.  I was thinking that these are all defined in advance by the Music Producers.  We could ask them.  This could also be crowd sourced...anyway this was just a long shot idea...

**Comment by mpogue2** [comment]
Also could pay attention to MP3 ID3 tag metadata, if there were a Type field.  Some people are inserting their own.  We should make this an industry standard (I've started a short document on this...)
